resources/pod.py

```diff
@@ -275,3 +278,103 @@ def run_io_and_verify_mount_point(pod_obj, bs='10M', count='950'):
     mount_point = mount_point.split()
     used_percentage = mount_point[mount_point.index('/var/lib/www/html') - 1]
     return used_percentage
+
+
+def get_pods_having_label(label, namespace):
+    """
+    Fetches pod resources with given label in given namespace
+
+    Args:
+        label (str): label which pods might have
+        namespace (str): Namespace in which to be looked up
+
+    Return:
+        dict: of pod info
+    """
+    ocp_pod = OCP(kind=constants.POD, namespace=namespace)
+    pods = ocp_pod.get(selector=label).get('items')
+    return pods
+
+
+def get_mds_pods(
+        mds_label=defaults.MDS_APP_LABEL,
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE
```

Comment:
I would set `namespace=None` and in the function set
```python
namespace = namespace or config.ENV_DATA['cluster_namespace'] 
```
This refer all the time to the cluster_namespace which should be one which is set from config for current cluster.

All the occurrences of this.